src/System.Console/tests/TermInfo.cs

```diff
@@ -58,6 +58,15 @@ public void VerifyInstalledTermInfosParse()
         Assert.True(foundAtLeastOne, "Didn't find any terminfo files");
     }
 
+    [Fact]
+    [PlatformSpecific(TestPlatforms.AnyUnix)] // Tests TermInfo
+    public void VerifyTermInfoSupportsNewAndLegacyNcurses()
+    {
+        MethodInfo readDbMethod = typeof(Console).GetTypeInfo().Assembly.GetType(TerminfoDatabaseType).GetTypeInfo().GetDeclaredMethods(ReadDatabaseMethod).Where(m => m.GetParameters().Count() == 2).Single();
+        readDbMethod.Invoke(null, new object[] { "xterm", "ncursesFormats" }); // This will throw InvalidOperationException in case we don't support the legacy format
+        readDbMethod.Invoke(null, new object[] { "screen-256color", "ncursesFormats" }); // This will throw InvalidOperationException if we can't parse the new format
```

Comment:
We'd also want to validate that the data we read is expected, at least spot checking some of the more important values from around the database.

Reply:
which values would you want me took for on the extended Strings? not sure which ones are the most valuable here.

Reply:
> which values would you want me took for on the extended Strings? not sure which ones are the most valuable here.

For example the color strings like those validated here:
https://github.com/dotnet/corefx/pull/27109/files#diff-90eedc08931731780b901ec0454101deR93